### PR TITLE
Add tracking to VEP tool run buttons

### PR DESCRIPTION
**Pull request by msmanoj**
Added tracking to the `Run` and `Run Instant VEP` buttons on the VEP tool page.

Sandbox URL:
http://ves-hx2-76.ebi.ac.uk:42228/Homo_sapiens/Tools/VEP?db=core

### CODE REVIEW

File: ga/htdocs/components/95_ga_configs.js [hunk 1]
```diff
@@ -758,6 +758,31 @@ Ensembl.GA.eventConfigs.push(
...
+
+  // Track Run VEP button
+  {
+    id              : 'RunVEPButton',
+    event           : 'click',
+    wrapper         : '.ajax.initial_panel',
+    selector        : 'input.run_button',
+    category        : 'RunVEPButton',
+    action          : 'Tools/VEP',
+    label           : ''
```
Comment (ens-ma7) [inline ga/htdocs/components/95_ga_configs.js]: i would advise to put something in the labels for both, in google analytics you can view events by label and we occasionally filter by label when viewing reports in google analytics. Otherwise it will be blank for these. Maybe something like 'VEP Instant Button' and 'Run VEP Button' will do.

Comment (ens-ma7) [reply ga/htdocs/components/95_ga_configs.js]: Also change category to 'Tools' for both. @ens-st3 what do you think about the name for the category, will 'tools' be good or is too generic?
